bravado/fido_client.py

```diff
@@ -1,19 +1,15 @@
 # -*- coding: utf-8 -*-
 import logging
-
+import requests
 import six
 
 if six.PY3:
     raise ImportError("The fido client is not yet supported in py3")
 
 import fido
-from bravado_core.param import stringify_body as param_stringify_body
```

Comment:
It seems that it is not used anywhere else:
https://github.com/Yelp/bravado/search?utf8=%E2%9C%93&q=stringify_body
https://github.com/Yelp/bravado-core/search?utf8=%E2%9C%93&q=stringify_body

Ticketed here https://github.com/Yelp/bravado-core/issues/91